include/z64collision_check.h

```diff
@@ -86,9 +86,14 @@ typedef struct ColliderElementDamageInfoAT {
     /* 0x05 */ u8 damage; // Damage
 } ColliderElementDamageInfoAT; // size = 0x08
 
+typedef enum HitBacklash {
+    HIT_BACKLASH_0,
+    HIT_BACKLASH_1 // Shock?
+} HitBacklash;
+
 typedef struct ColliderElementDamageInfoAC {
     /* 0x00 */ u32 dmgFlags; // Damage types that may affect this collider element as AC.
-    /* 0x04 */ u8 effect;  // Damage Effect (Knockback, Fire, etc.)
+    /* 0x04 */ u8 hitBacklash;  // The backlash type applied to other ATs touching this AC.
     /* 0x05 */ u8 defense; // Damage Resistance
     /* 0x06 */ Vec3s hitPos; // Point of contact
```

Comment:
Just brainstorming here, your comments on the struct members got me thinking:

Maybe this can be `backlashType` and the `at` one can be `receivedBacklash` , or something similar?
I think names like this can make the "give and take" more obvious without needing to rely on "at" being in the name

Reply:
what about the enum? still `HitBacklash` / `HIT_BACKLASH_` ?

Reply:
I guess we need to decide if "hit" is really necessary in the name.
If it is deemed necessary, then i guess my suggestions above should be `hitBacklashType` and `hitBacklashRecieved`

Reply:
idk about "received" at the end of things, as everything in CollisionCheckInfo is received by the actor

like, we could also name receivedDamageReaction and receivedHitEffect ("trait") with received, I don't think it adds much in the end. The point being to distinguish those three fields damageReaction / backlash / hitEffect